cmd/main.go

```diff
@@ -164,18 +169,30 @@ func main() {
 	signal.Notify(c, os.Interrupt)
 
 	// subscribing to nats
-	natsClient, err := nats.New("<server-url>")
+	natsClient, err := nats.New("nats://127.0.0.1:4222")
 	if err != nil {
 		logrus.Printf("Nats client create error %v", err)
 	} else {
-		err = natsClient.Subscribe("cluster")
+		msgch := make(chan *broker.Message)
+		err = natsClient.SubscribeWithChannel("cluster", "cluster-queue", msgch)
 		if err != nil {
 			logrus.Printf("Error subscribing to cluster %v", err)
 		}
-		err = natsClient.Subscribe("istio")
+		err = natsClient.SubscribeWithChannel("istio", "istio-queue", msgch)
 		if err != nil {
 			logrus.Printf("Error subscribing to istio %v", err)
 		}
+
+		go func() {
+			for {
+				msg := <-msgch
+				objectJSON, _ := json.Marshal(msg.Object)
+				var object model.Object
+				_ = utils.Unmarshal(string(objectJSON), &object)
+				// persist the object
+				log.Printf("%+v\n", object)
```

Comment:
@kumarabd, do we have a flag for running Meshery in a global verbose mode?

Reply:
@mgfeller such a mode might be great fodder for MeshKit.

Reply:
@leecalcote  where do u mean we would need the verbose? Because the logging that we are doing here is temporary and meshkit has DEBUG logs which can be turned ON/OFF for verbosity.

Reply:
Should we toss these log statements behind the debug logger? Or are these statements transient and to be removed in a shortly forthcoming PR?

Reply:
@leecalcote Meshkit logger is not yet been used above at the moment. Yes these will be removed in the forthcoming PRs.